spec/controllers/embed_controller_spec.rb

```diff
@@ -20,11 +20,15 @@
 
     before do
       Fabricate(:embeddable_host)
+      SiteSetting.stubs(:embed_link_to_top).returns(true)
```

Comment:
No worries. I can change that. There's quite a few [examples](https://github.com/discourse/discourse/blob/master/spec/controllers/application_controller_spec.rb#L180) of stubbing out settings already in the code base. Would that be a different use case?